src/reactive-membrane.ts

```diff
@@ -37,6 +43,14 @@ export interface ObservableMembraneInit {
     valueIsObservable?: ReactiveMembraneObservableCallback;
 }
 
+export interface MembraneProxyHandler extends ProxyHandler<any> {
```

Comment:
formalization of the api of the handler, so we can pass it around into abstraction pieces rather than passing internal references

Reply:
maybe to the base handler as well? or this is in fact the base... will do